.api-version

```diff
@@ -1 +1 @@
-5.2.2
+5.3.0
```

Comment:
This should only be done for enteprise releases, as everything but the changelog is meant to represent the state of enterprise.